src/app/ResourceManager/LocalizationHelpers.cs

```diff
@@ -136,13 +138,11 @@ public static string ProcessSubmoduleStatus(IGitModule module, GitSubmoduleStatu
                 if (gitModule.IsValidGitWorkingDir()
                     && commitDataManager.GetCommitData(status.OldCommit.ToString(), cache: true) is CommitData c)
                 {
-                    oldCommitData = commitDataManager.GetCommitData(status.OldCommit.ToString(), cache: true);
                     oldCommitData = c;
 
                     sb.AppendLine("\t\t\t" + GetRelativeDateString(DateTime.UtcNow, oldCommitData.CommitDate.UtcDateTime) + " (" +
                                   GetFullDateString(oldCommitData.CommitDate) + ")");
-                    string[] lines = oldCommitData.Body.Trim(Delimiters.LineFeedAndCarriageReturn).Split(Delimiters.LineFeedAndCarriageReturn, StringSplitOptions.None);
-                    foreach (string line in lines)
+                    foreach (string line in oldCommitData.Body.Replace("\r\n", "\n").Split(Delimiters.LineFeed, StringSplitOptions.None))
```

Comment:
Have you removed `Trim` on purpose?

Reply:
Yes, presentation should be as in the message, just indented by two tabs